ui-data/html/analysis.html

```diff
@@ -100,7 +108,9 @@
                   "so" in parsedJson &&
                   "s" in parsedJson &&
                   "bo" in parsedJson &&
-                  "v" in parsedJson) {
+                  "v" in parsedJson &&
+                  "dl" in parsedJson &&
```

Comment:
So since we are adding some extra properties to the json string we are expecting from the clipboard I don't know if we need to increment the version number of the definition.

At a minimum we should make them optional properties when pasting in.